command/bridge/deposit/erc20/deposit_erc20.go

```diff
@@ -197,6 +195,8 @@ func runCommand(cmd *cobra.Command, _ []string) {
 					return fmt.Errorf("receiver: %s, amount: %s", receiver, amount)
 				}
 
+				blockNumbers = append(blockNumbers, receipt.BlockNumber)
```

Comment:
Could it happen, since there is no lock, that two go routines try to append at the same time, and the results for one of them get overwritten by the other one? Also, since there are multiple go routines, could it happen that the same block gets added to the slice multiple times, if different transactions enter the same block?

Reply:
Good catch. :monocle_face: :slightly_smiling_face: 

Fixed https://github.com/0xPolygon/polygon-edge/pull/1584/commits/79db3787ad127c4558abdd76c0369c6185e5c41a